app/models/song.rb

```diff
@@ -1,2 +1,5 @@
 class Song < ApplicationRecord
```

Comment:
Maybe you add validation about length fields? For example, the name should have length more 3 chars an etc..